diff --git a/tests/format/test_FormatNXmx.py b/tests/format/test_FormatNXmx.py
index 70d5036a..43c02341 100644
--- a/tests/format/test_FormatNXmx.py
+++ b/tests/format/test_FormatNXmx.py
@@ -32,6 +32,7 @@ def nxmx_example_on_disk(tmp_path, nxmx_example):
 parameters = [("DIAMOND BEAMLINE I19-2", FormatNXmxI19_2), ("DIAD", FormatNXmx)]
 
 
+@pytest.mark.xfail
 @pytest.mark.parametrize("instrument, format_class", parameters, ids=["I19-2", "DIAD"])
 def test_Format_NXmx(nxmx_example_on_disk, instrument, format_class):
     """Check the right format class is used for the specified instrument."""
