diff --git a/tests/command_line/test_rs_mapper.py b/tests/command_line/test_rs_mapper.py
index 8fdeffd88..f10d19e19 100644
--- a/tests/command_line/test_rs_mapper.py
+++ b/tests/command_line/test_rs_mapper.py
@@ -14,6 +14,7 @@ def test_rs_mapper(dials_data, tmp_path):
     result = subprocess.run(
         [
             shutil.which("dials.rs_mapper"),
+            "nproc=1",
             dials_data("centroid_test_data", pathlib=True)
             / "imported_experiments.json",
             'map_file="junk.ccp4"',
@@ -43,6 +44,7 @@ def test_multi_panel(dials_regression: pathlib.Path, tmp_path):
     result = subprocess.run(
         [
             shutil.which("dials.rs_mapper"),
+            "nproc=1",
             image,
             'map_file="junk.ccp4"',
         ],
@@ -75,7 +77,7 @@ def test_masked(dials_data, tmp_path):
         capture_output=True,
     )
     result = subprocess.run(
-        [shutil.which("dials.rs_mapper"), "imported.expt", "map_file=masked.ccp4"],
+        [shutil.which("dials.rs_mapper"), "nproc=1", "imported.expt", "map_file=masked.ccp4"],
         cwd=tmp_path,
         capture_output=True,
     )
@@ -86,6 +88,7 @@ def test_masked(dials_data, tmp_path):
     result = subprocess.run(
         [
             shutil.which("dials.rs_mapper"),
+            "nproc=1",
             "imported.expt",
             "map_file=unmasked.ccp4",
             "ignore_mask=True",
